libs/blog-contracts/articles/src/lib/articles.ts

```
@@ -78,6 +78,27 @@ export type SeoMetaData = Pick<
+export interface ArticleTranslation {
+  locale: string;
+  slug: string;
+}
+
+export const enum ArticleStatus {
+  Publish = 1,
+  Future = 2,
+  Draft = 3,
+  Private = 4,
+  Pending = 5,
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)